pycbc/workflow/injection.py

```diff
@@ -147,12 +147,9 @@ def cut_distant_injections(workflow, inj_file, out_dir, tags=None):
     return node.output_files[0]
 
 def inj_to_hdf(workflow, inj_file, out_dir, tags=None):
-    """ Convert injection file to hdf format if not already one
+    """ Convert injection file to hdf format, if it is already one,
+    this just makes a copy
```

Comment:
Docstring naming convention says that line 1 should be a single sentence.

So this should really be:

```
""" Convert injection file to hdf format.

If the file is already PyCBC HDF format, this will just make a copy.
"""